config/dictionaries/resource.json

```
@@ -438,6 +438,52 @@
       }
     ]
   },
+  {
+    "id": "device_type",
+    "name": "Device type",
+    "fields": [
+      "id",
+      "slug",
+      "name",
+      "is_private",
+      "is_of__cpu_architecture",
+      "belongs_to__device_family"
+    ],
+    "examples": [
+      {
+        "id": "device-type-by-id",
+        "summary": "Get the details of a device type by ID.",
+        "description": "The bearer token is optional for targeting public device types, but is required for private ones.",
```

```suggestion
        "summary": "Get device type by ID",
        "description": "You can add a bearer token to the request for target private device types",
```

By ID it be better to specify exactly what ID we need for this request. Explicitly mentioning is better.
If you like the description, then please change that for all entries.